lib/streamlit/server/server.py

```diff
@@ -79,11 +79,11 @@
 from streamlit.server.routes import MediaFileHandler
 from streamlit.server.routes import MessageCacheHandler
 from streamlit.server.routes import StaticFileHandler
-from streamlit.server.server_util import MESSAGE_SIZE_LIMIT
 from streamlit.server.server_util import is_cacheable_msg
 from streamlit.server.server_util import is_url_from_allowed_origins
 from streamlit.server.server_util import make_url_path_regex
 from streamlit.server.server_util import serialize_forward_msg
+from streamlit.server.server_util import _get_max_message_size_bytes
```

Comment:
nit: one small thing that I must have missed last review -- if this is being used outside of `server_util`, by convention the trailing underscore should be removed